xbmc/video/VideoInfoScanner.cpp

```diff
@@ -1964,14 +1966,7 @@ namespace KODI::VIDEO
       const ScraperPtr& info(scraper);
       std::unique_ptr<IVideoInfoTagLoader> loader;
       if (useLocal)
-      {
-        loader.reset(CVideoInfoTagLoaderFactory::CreateLoader(item, info, false));
-        if (loader)
-        {
-          // no reset here on purpose
```

Comment:
Looks like this instance does not reset the InfoTag, unlike the other callers but ReadInfoTag() always resets it.